src/data-dictionary/events/NrAiIncident/closeCause.md

```diff
@@ -7,4 +7,19 @@ events:
 
 If applicable, what caused the incident to close.
 
-Available values: `CONDITION_DELETED`, `POLICY_DELETED`, `EVALUATOR`, `EXPIRED`, `CONDITION_MODIFIED`, `LOSS_OF_SIGNAL`, `USER`, `TARGET_REMOVED`, `INCIDENT_WORKFLOW_INTEGRATION`, and `CONDITION_DISABLED`.
+Available values: 
+`CONDITION_DELETED`: condition was deleted.
+`CONDITION_DISABLED` condition was disabled.
+`CONDITION_MODIFIED` condition was [edited](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#evaluation-resets)
+`EVALUATOR`: natural close because the metric has ceased to breach the threshold.
```

Comment:
Think it'd be nice if this had more active voice

Reply:
done!